features/601-asn-update-in-nextui/test.feature

```diff
@@ -65,3 +65,43 @@ Feature: 601 - ASN Update / Correction in the Next UI
 			And I see the correction for "ASN" to "11/01ZD/01/00000410836V"
 		Then I submit the record
 			And I reload until I don't see "(Submitted)"
+
+	@NextUI
+	@ExcludeOnLegacyUI
+	Scenario: Displays relevant resolution status when exception resolved manually
+		Given I am logged in as "supervisor"
+			And I view the list of exceptions
+		Then I see exception "HO100206" in the exception list table
+		When I open the record for "SEXOFFENCE TRPRFOUR"
+			Then I manually resolve the record
+			Then I see exceptions resolution status as "Manually Resolved"
```

Comment:
What if you leave the case and come back to it?

Reply:
Added a test to cover that scenario